packages/logger/src/tests/logger.ts

```diff
@@ -95,17 +96,17 @@ describe('Basic logger test', () => {
       getLogger().error(message);
       expect(consoleSpy).toHaveBeenCalledWith<[string]>(
         expect.stringMatching(
-          new RegExp(`^{.*\\"serviceContext\\":{\\"service\\":\\"@tree-house/logger\\",\\"version\\":\\".*-${process.env.NODE_ENV}\\"}.*}\n$`),
+          new RegExp(
+            `^{.*\\"serviceContext\\":{\\"service\\":\\"@tree-house/logger\\",\\"version\\":\\".*-${process.env.NODE_ENV}\\"}.*}\n$`,
+          ),
```

Comment:
@gorjuspixels For some reason this test starts failing after either upgrading yarn or aligning TS with the root tsconfig.json. Any clue why? The error is the following:

```
 Expected: StringMatching /^{.*\"serviceContext\":{\"service\":\"@tree-house\/logger\",\"version\":\".*-test\"}.*}\n$/
@tree-house/logger:     
Received: "{\"level\":\"🔥 error\",\"serviceContext\":{\"version\":\"undefined-test\"},\"namespace\":\"test\",\"eventTime\":\"1970-01-01T00:00:00.000Z\",\"context\":{},\"message\":\"Dolore voluptas voluptatem.\"}
```

Reply:
ps: I know I updated the `debug` package but the error occurred before this so that's not the reason